### DISCUSSION TITLE
UAC for RustDesk? Which setting to disable it?

### DISCUSSION THREAD

**Discussion by nicedevil007** [rustdesk/rustdesk-server-pro#264] [Q&A]
Heya, I have an other question. We used AnyDesk/Teamviewer in the past and didn't need any UAC during the connection to any client. With rustdesk it seems like it needs any kind of administrative approval for a setting that I can connect to my clients.

Can you tell me which setting has to be disabled to get rid of this?

Here are my Advanced custom settings for my custom client:

```java
access-mode=custom
enable-keyboard=Y
enable-clipboard=Y
enable-file-transfer=Y
enable-audio=N
enable-tunnel=N
enable-remote-restart=N
enable-record-session=N
allow-remote-config-modification=N
enable-lan-discovery=N
direct-server=N
allow-auto-disconnect=N
auto-disconnect-timeout=60
allow-only-conn-window-open=N
theme=dark
lang=de
allow-auto-record-incoming=N
allow-always-software-render=N
allow-linux-headless=N
enable-hwcodec=Y
show-monitors-toolbar=Y
show-quality-monitor=N
disable-audio=Y
enable-file-copy-paste=Y
disable-clipboard=N
lock-after-session-end=N
privacy-mode=N
whitelist=,
enable-abr=Y
allow-remove-wallpaper=N
enable-open-new-connections-in-tabs=Y
peer-card-ui-type=2
peer-sorting=Remote Host
view-only=N
collapse-toolbar=Y
show-remote-cursor=Y
follow-remote-cursor=Y
follow-remote-window=Y
zoom-cursor=Y
view-style=adaptive
enable-block-input=N
```

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/17103076/4e3ad608-4520-415c-b6e6-748efacc4c51)

**Comment by rustdesk** [comment]
"seems like", screenshot about this?

**Reply by nicedevil007** [reply]
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/17103076/50d8c2ef-1621-49e7-8b70-13b5d7133714)

I mean this one, this always shows up on the client, when I hit the connect button.

**Reply by rustdesk** [reply]
Your case seems quite special. if possible, please upload a video.

> this always shows up on the client, when I hit the connect button.

where does it show up? on the remote side, or local side (where you click on connect button).

**Reply by nicedevil007** [reply]
I'm the user on `PC one`, that want to connect to a client, we call it `PC two`. I enter the ID of the `PC two` and hit the connect button. Then this window appears on the `PC two`.

**Reply by rustdesk** [reply]
@ZetaWaves coud you help confirm if you have ever seen such UAC (`cmd.exe`)?  We do not call `cmd.exe` during connection.

**Reply by 21pages** [reply]
@nicedevil007 Could you help to debug this? 
1. On the controlled side, is rustdesk installed or portable? What if installed if portable is used?
2. If you unremember the password, and click to accept the connection on the controlled end, what is the order of occurrence of a, b and c? 
* a. A window for receiving connections pops up in the upper right corner of the desktop of the controlled end
* b. UAC pops up
* c. The control end displays the screen
3. Will UAC pop up if you start a file transfer connection?

**Reply by ZetaWaves** [reply]
> @ZetaWaves coud you help confirm if you have ever seen such UAC (`cmd.exe`)? We do not call `cmd.exe` during connection.

I have not seen a UAC prompt that used cmd.exe. 

It seems this user has the client installed. The only time I’ve seen cmd called is if client is being uninstalled.

What happens if user clicks No to the UAC prompt? 

What settings does the controlling-end use? Are they attempting to disable user input or use Privacy Mode? Those could cause a need for elevation. 

Anything else, seems may be an issue with the user or the user’s settings and not RustDesk itself.
